protocol/x/clob/memclob/memclob.go

```diff
@@ -1132,12 +1132,12 @@ func (m *MemClobPriceTimePriority) RemoveAndClearOperationsQueue(
 			otpOrderHash := operation.GetShortTermOrderPlacement().Order.GetOrderHash()
 
 			// If the order exists in the book, remove it.
-			// Else if the order is a Short-Term order, since it's no longer on the book or operations
-			// queue we should remove the order hash from `ShortTermOrderTxBytes`.
+			// Else, since the Short-Term order is no longer on the book or operations queue we
+			// should remove the order hash from `ShortTermOrderTxBytes`.
 			existingOrder, found := m.openOrders.getOrder(ctx, otpOrderId)
 			if found && existingOrder.GetOrderHash() == otpOrderHash {
 				m.mustRemoveOrder(ctx, otpOrderId)
-			} else if otpOrderId.IsShortTermOrder() {
+			} else {
```

Comment:
This shouldn't be necessary since all of these operations should be short-term orders